### PR TITLE
CDK: erigon docs

### PR DESCRIPTION
**Pull request by kmurphypolygon**

### CODE REVIEW

**Review by vcastellm — CHANGES_REQUESTED** [review]
Let's put this on hold as there are some modifications needed to make this more friendly

**Comment by kmurphypolygon** [comment]
No worries, moving into draft state.

File: docs/cdk/erigon/chain-config.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+1. Ensure the chain name starts with the word `dynamic` e.g. `dynamic-mynetwork`.
+
+2. Create the following files for dynamic configs. The examples for Cardona are in `zk/examples/dynamic-configs` and can be edited as required:
+
+    - `dynamic-{network}-allocs.json` - the allocs file.
+    - `dynamic-{network}-chainspec.json` - the chainspec file.
+    - `dynamic-{network}-conf.json` - an additional configuration file.
+    - `dynamic-{network}.yaml` - the run config file for erigon.  
+    
+    You can use any of the example yaml files at the root of the repo as a base and edit as required, but ensure the `chain` field is in the format `dynamic-{network` and matches the names of the config files above.
```
Comment (vcastellm) [inline docs/cdk/erigon/chain-config.md]:
```suggestion
    You can use any of the example yaml files at the root of the repo as a base and edit as required, but ensure the `chain` field is in the format `dynamic-mynetwork` and matches the names of the config files above.
```

**Review by vcastellm — APPROVED** [review]
LGTM!

File: docs/cdk/erigon/chain-config.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+        - `zkevm.address-sequencer` => `create_rollup_output.json` => `sequencer`
+        - `zkevm.address-zkevm` => `create_rollup_output.json` => `rollupAddress`
+        - `zkevm.address-admin` => `deploy_output.json` => `admin`
+        - `zkevm.address-rollup` => `deploy_output.json` => `polygonRollupManagerAddress`
+        - `zkevm.address-ger-manager` => `deploy_output.json` => `polygonZkEVMGlobalExitRootAddress`
+
+4. Mount the directory containing the config files on a Docker container. For example `/dynamic-mynetwork`.
+
+5. To use the new config when starting erigon, use the `--cfg` flag with the path to the config file e.g. `--cfg="/dynamic-mynetwork/dynamic-mynetwork.yaml"`.
```
Comment (revitteth) [inline docs/cdk/erigon/chain-config.md]: This should be --config flag :)

Comment (kmurphypolygon) [reply docs/cdk/erigon/chain-config.md]: Thank you. Updated.

**Pull request merged**